.github/workflows/stale.yml

```diff
@@ -0,0 +1,27 @@
+name: "Close stale Issues and PRs"
+on:
+  schedule:
+    - cron: "30 0 * * *"
```

Comment:
This is scheduled to run 30 minutes after lock cron job.

https://crontab.guru/#30_0_*_*_*